Sources/Slacket/Parsers/ParserType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ParserType.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+import Kitura
+
+// To define models that can be parsed
+protocol Parsable {}
+
+// To define parsers
+protocol ParserType {}
+
+protocol ParserDecoder: ParserType {
+    
+    func parse(body: ParsedBody) -> Parsable
```

Comment:
I don't understand why `SlackCommandDecoder` would return `SlackCommandDecoder`... ?
My idea was to have a base protocol, something that concrete Parsers could conform to. But that is just a proposition .How would you introduce abstraction here?